Test/git-issues/git-issue-1963b.dfy.expect

```diff
@@ -1,2 +1,2 @@
-git-issue-1963b.dfy(12,15): Error: Refinement cannot change the constructors of a `datatype`.  To refine `D`, either omit this `...` or omit the `=` sign and the datatype constructors.
+git-issue-1963b.dfy(12,15): Error: Refinement cannot change the constructors of a datatype. To refine 'D', either omit this '...' or omit the '=' sign and the datatype constructors.
```

Comment:
Similar issue as in `1963a`.

Reply:
Here it turns out the parser is overly strict -- put this bug in Issue 3292

Reply:
In https://github.com/dafny-lang/dafny/issues/3292 you say that the error messages are confusing, and that they are being fixed with this PR. But I can't seem to find any changes for`1963a` and `1963b` that address the issue?

Reply:
After discussion with Rustan, the code is correct. I improved the error message and now have correct the expect files OK.